project_name/settings/development.py

```diff
@@ -1,18 +1,6 @@
 import os
 
 from {{ project_name }}.settings.base import *  # noqa: F401, F403
-from {{ project_name }}.settings.base import PLATFORM_ACCOUNTS
 
 
 os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "1"
-PLATFORM_ACCOUNTS.update(
```

Comment:
What about in `ci.py`? will `PLATFORM_ACCOUNTS` not being defined throw any errors?

Reply:
`ci.py` will use the values defined in `base.py`, but in CI responses are all mocked when testing, so it shouldn't matter.